engine/runtime-with-polyglot/src/test/java/org/enso/interpreter/test/instrument/VerifyJavaScriptIsAvailableTest.java

```diff
@@ -41,11 +41,13 @@ public static void initEnsoContext() {
   public static void closeEnsoContext() throws Exception {
     ctx.close();
 
+    /* XXX: Is this needed for startup?
```

Comment:
Remove the commented code. I would most likely get rid of that in https://github.com/enso-org/enso/pull/8467 anyway

Reply:
Removed in ef1c79825a